test/services/challenges_repository.js

```diff
@@ -183,38 +140,55 @@ describe('Challenges repository', () => {
       configWrapperMock = {
         challengeDuration: sinon.stub().resolves(challengeDuration)
       };
-      challengesRepository = new ChallengesRepository(challengeWrapperMock, configWrapperMock);
-      sinon.spy(challengesRepository, 'extractChallengeFromEvent');
-      sinon.stub(challengesRepository, 'filterOutFinishedChallenges').returns(notFinishedEvents);
-      sinon.stub(challengesRepository, 'sortChallenges').returns(sortedEvents);
+      activeChallengesCacheMock = {
```

Comment:
I would prefer using the real thing in this test (`new ActiveChallengesCache()`).

Reply:
Why? `ActiveChallengesCache ` is tested separately.